contracts/DomainRegistar.sol

```diff
@@ -4,35 +4,25 @@ pragma solidity ^0.8.24;
 // Uncomment this line to use console.log
 // import "hardhat/console.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
-
+import "./DomainUtils.sol";
 
 error AccessDenied(string description);
 error NotEnoughFunds(uint provided, uint required);
 error DuplicateDomain();
-error TopLevelDomainsOnly();
+
 
 /**
  * @title A simple contract for top-level domain registration
  * @author Me
  */
 contract DomainRegistar is Initializable {
-    struct DomainEntry {
-        /// @notice Address of the account that deployed the contract
-        address payable owner;
-
-        /// @notice Price payed for domain registration
-        uint weiDomainPrice;
```

Comment:
line 24  and other line with same logic 
    uint -> as a good practice, it is better to use uint256
    // - 1 point